app/assets/stylesheets/organisms/_form-card.scss

```diff
@@ -120,3 +120,82 @@
     }
   }
 }
+
+.form-card__target {
+  text-align: center;
+  border: solid 5px transparent;
+  font-weight: bold;
+  color: $color-grey;
+  padding: 3rem;
+  margin-bottom: 3rem;
+  min-height: 5rem;
+  cursor: pointer;
+
+  &.dz-drag-hover {
+    border: dashed 5px $color-light-grey;
+  }
+
+  i.icon-file_upload {
+
+  }
+}
+
+.form-card__documents {
+  &:empty { display: none; }
+
+  .dz-preview {
```

Comment:
Makes sense, although I wonder if we can alias `dz` to `shubox` somehow to make this clearer? Not a huge deal, just speculating.